README.md

```diff
@@ -1,6 +1,6 @@
 # Rakudo Star
 
-[![Build Status](https://travis-ci.org/raku/docker.svg?branch=master)](https://travis-ci.org/perl6/docker)
+[![Build Status](https://travis-ci.org/raku/docker.svg?branch=master)](https://travis-ci.org/raku/docker)
 
```

Comment:
Can yo please capitalize? It's probably OK, but it's better if it's got the same URL as this repo.

Reply:
Done.  I've just pushed an update to the PR.